src/act.obj.cpp

```diff
@@ -3659,6 +3682,11 @@ ACMD(do_ready)
     GET_HOLSTER_READY_STATUS(obj) = 0;
     return;
   } else {
+    //Check if we have any fingertip compartments that are readied
+    for (struct obj_data *cyber = ch->cyberware; cyber; cyber = cyber->next_content)
+      if (GET_OBJ_VAL(cyber, 0) == CYB_FINGERTIP && GET_HOLSTER_READY_STATUS(cyber))
```

Comment:
It does. I made sure that the value is not used for anything for the fingertip compartment before going with this. There was no point for another macro and extra code since the value was free.